python/pyspark/mllib/util.py

```diff
@@ -169,6 +175,32 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to
+        the end of the input vector.
+        """
+        vec = _convert_to_vector(data)
+        if isinstance(vec, SparseVector):
+            if _have_scipy:
```

Comment:
Oh, I just realized that convert_to_vector already handles scipy for you.  You may assume that you have either a SparseVector or a DenseVector.  No need to handle scipy here.